web-wallet/src/lib/containers/MigrateContract/MigrateContract.svelte

```diff
@@ -356,11 +343,20 @@
         />
       {:else}
         <div class="migrate__execute">
-          <Icon path={mdiCheckDecagramOutline} />
-          <p>Migration completed successfully!</p>
+          <div class="migrate__execute-approval">
+            <Icon path={mdiCheckDecagramOutline} size="large" />
+            <p>Migration request accepted!</p>
+          </div>
+          <Banner title="Migration Request Accepted" variant="info">
+            <p>
+              The migration request has now been accepted on chain. We will
+              process the request in the background. Receiving your DUSK might
+              take some time. Check back again later.
```

Comment:
I think the `check back again later` might confuse users in thinking they have to check the `Migrate` page again.

Maybe something like this would clarify the ambiguity?
```suggestion
              take some time. Check your wallet balance again later.
```

Reply:
Done!